apps/dashboard/app/models/support_ticket_validator.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+# Support ticket validator
+# Validates all the support ticket attributes based on their metadata.
+# Supported validations:
+#  - required
+#  - email
+#  - attachments
+module SupportTicketValidator
```

Comment:
Can this move to `app/models/concerns`?  I think that's the pattern we should follow for modules like this.

Reply:
Updating...